pensieve/cli.py

```diff
@@ -96,12 +97,22 @@ def run(project_id, dataset_id, date, experiment_slug, dry_run, config_file):
         # run analysis for specific experiment
         active_experiments = active_experiments.with_slug(experiment_slug)
 
+    # get experiment-specific external configs
+    external_configs = ExternalConfigCollection.from_github_repo()
+
     # calculate metrics for experiments and write to BigQuery
     for experiment in active_experiments.experiments:
         spec = default_spec_for_experiment(experiment)
+
         if config_file:
             custom_spec = AnalysisSpec.from_dict(toml.load(config_file))
             spec.merge(custom_spec)
+
+        external_experiment_config = external_configs.spec_for_experiment(experiment.normandy_slug)
+
+        if external_experiment_config:
+            spec.merge(external_experiment_config)
```

Comment:
this means that an external experiment config is higher-priority than a config file specified on the CLI, which seems backwards to me; I think it probably makes sense to skip the external config if a secret CLI config is specified, since otherwise it makes debugging more confusing. wdyt?

(same sitch in `rerun`)

Reply:
Yes, that sounds reasonable.